Airwallex/Core/Sources/AWXFloatingCardTextField.h

```diff
@@ -0,0 +1,20 @@
+//
+//  AWXFloatingCardTextField.h
+//  Core
+//
+//  Created by Hector.Huang on 2022/11/10.
+//  Copyright © 2022 Airwallex. All rights reserved.
+//
+
+#import "AWXWidgets.h"
+
+/**
+ A customized view for card number
+ */
+@interface AWXFloatingCardTextField : AWXFloatingLabelTextField
+
+@property (nonatomic, copy) NSArray *cardBrands;
```

Comment:
I can tell you it's an array of enum cases (which is NSUInteger), whereas in objective c I can't do either NSArray<NSUInteger> or NSArray<AWXBrandType> because of the type in <> needs to be an object type rather than primitives.